openfisca_nsw_safeguard/variables/ESS_PDRS_Estimator/HVAC1/certificate_estimation/HVAC1_ESC_calculation.py

```diff
@@ -95,7 +95,28 @@ def formula(buildings, period, parameters):
                         (heating_capacity * equivalent_heating_hours) / baseline_ACOP
                     ])
 
+
+class HVAC1_THEC(Variable):
```

Comment:
can we change this variable name to something more informative?

like HVAC1_thec_or_annual_reference_heating

Reply:
Okay will do - hold the line